src/analysis/special_functions/pow.lean

```diff
@@ -601,6 +601,78 @@ end sqrt
 
 end real
 
+section measurability_real
+
+lemma real.measurable_rpow : measurable (λ p : ℝ × ℝ, p.1 ^ p.2) :=
+begin
+  have h_prod : {p : ℝ × ℝ | p.1 = 0} = set.prod {(0:ℝ)} set.univ,
+  { ext,
+    rw [set.mem_set_of_eq, set.mem_prod, set.mem_singleton_iff],
+    simp, },
+  have h_meas : is_measurable {p : ℝ × ℝ | p.1 = 0},
+  { rw [h_prod, is_measurable_prod],
+    simp, },
```

Comment:
```suggestion
  have h_meas : is_measurable {p : ℝ × ℝ | p.1 = 0} :=
    (is_closed.preimage continuous_fst is_closed_singleton).is_measurable,
```

Reply:
That is indeed much more efficient. Thanks.